code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -47,6 +47,8 @@
 							ABNORMALITY_WORK_ATTACHMENT = list(50, 55, 60, 65, 70),
 							ABNORMALITY_WORK_REPRESSION = list(50, 55, 60, 65, 70)
 							)
+	/// Work Types and corresponding their attributes
+	var/list/work_attribute_types = WORK_TO_ATTRIBUTE
```

Comment:
Shouldn't this be a list?

Reply:
It IS a list, WORK_TO_ATTRIBUTE is a list defined in attributes.dm
```
#define WORK_TO_ATTRIBUTE list(\
							ABNORMALITY_WORK_INSTINCT = FORTITUDE_ATTRIBUTE,\
							ABNORMALITY_WORK_INSIGHT = PRUDENCE_ATTRIBUTE,\
							ABNORMALITY_WORK_ATTACHMENT = TEMPERANCE_ATTRIBUTE,\
							ABNORMALITY_WORK_REPRESSION = JUSTICE_ATTRIBUTE,\
							)
```

Reply:
Oh what the fuck